scripts/qa_crowbarsetup.sh

```diff
@@ -1005,6 +1020,9 @@ EOF
         else
             complain 18 "You requested a HA setup but for this combination ($cloudsource : $slesdist) no HA setup is available."
         fi
+        if iscloudver 6plus; then
```

Comment:
Also, this function is already way too long.  If there's an opportunity to refactor into smaller pieces whilst making this change, please take it :)